test/Tester/ClientConnectionTests/GatewayConnectionTests.cs

```diff
@@ -48,16 +49,24 @@ public override TestCluster CreateTestCluster()
         {
             var options = new TestClusterOptions(1)
             {
-                ClientConfiguration =
-                {
-                    GatewayProvider = ClientConfiguration.GatewayProviderType.Custom,
-                    CustomGatewayProviderAssemblyName = "Tester",
-                    GatewayListRefreshPeriod = TimeSpan.FromMilliseconds(100)
```

Comment:
you lost this configuration in the transformation, is it not needed?

Reply:
no I didn't. take a look on the right :) ClientConfiguration = {GatewayListRefreshPeriod = TimeSpan.FromMilliseconds(100)}